### PR TITLE
[BUILD-1061] - Favicon Tab Change

### PR DESCRIPTION
**Pull request by FayeMao**
# Description
[BUILD-1061](https://app.clickup.com/t/9009201449/BUILD-1061)

In this PR, I changed the favicon and changed the tab names.

# To run
1. Go to branch
2. `yarn install`
3. `yarn start`
4. Go to different page types (e.g. homepage, product, catalog, etc.) and see the tab. The tab name should be `Dynamic to be Design Studio | [page name or product name]`. Check to see if the favicon is updated (should be an orange circle)

### CODE REVIEW

**Comment by kirstenjvd** [comment]
Task linked: [BUILD-1061 <Title> text change and Favicon](https://app.clickup.com/t/9009201449/BUILD-1061)

File: src/images/favicon-32x32.png
Comment (FayeMao) [inline src/images/favicon-32x32.png:1]: This doesn't actually get added anywhere (that I know of)

Comment (FayeMao) [reply src/images/favicon-32x32.png:1]: LOL jokes it does change the favicon

File: gatsby-config.js
Comment (kirstenjvd) [inline gatsby-config.js:1]: Can you change this just to "Design Studio"  vs "dynamic to be Design Studio" - otherwise it looks great!

**Review by kirstenjvd — CHANGES_REQUESTED** [review]

**Pull request merged**